packages/jest-haste-map/src/haste_fs.js

```diff
@@ -40,6 +43,11 @@ export default class HasteFS {
     return this._files.has(file);
   }
 
+  follow(file: Path): Path {
+    const link = this._links[file];
+    return link ? link[0] || (link[0] = fs.realpathSync(file)) : file;
```

Comment:
could use `realpath-native`

Reply:
👍

Split the side-effect in two as well.